README.md

```diff
@@ -15,10 +15,33 @@ To add OpenSSL in your QMake project, append the following to your `.pro` projec
 android: include(<path/to/android_openssl/openssl.pri)
 ```
 
-To add OpenSSL in your CMake project, append the following to your project's `CMakeLists.txt` file, anywhere after the qt_add_executable() call for your target application:
+To add OpenSSL in your CMake project, append the following to your project's `CMakeLists.txt` file:
 
 ```
 if (ANDROID)
-    include(<path/to/android_openssl/CMakeLists.txt)
+    FetchContent_Declare(
+      android_openssl
+      DOWNLOAD_EXTRACT_TIMESTAMP true
+      URL      https://github.com/KDAB/android_openssl/archive/refs/heads/master.zip
+#      URL_HASH MD5=c97d6ad774fab16be63b0ab40f78d945 #optional
+    )
+    FetchContent_MakeAvailable(android_openssl)
```

Comment:
I feel like this part could a sub-section of its own

Reply:
I considered it, but i think it only makes the readme unnecessarily longer. Though I added a subsections for qmake and cmake.